tutorialtool/README.md

```diff
@@ -0,0 +1,30 @@
+# Teacher Tool
+
+## Localhost testing
+
+To test the Teacher Tool locally:
+
+1. Ensure your pxt repo is up to date and has been built recently.
+2. In a command shell, in the `pxt` repo, cd into the `tutorialtool` folder and start the Teacher Tool dev server: `npm run start`. This will *not* open a browser window.
```

Comment:
```suggestion
2. In a command shell, in the `pxt` repo, cd into the `tutorialtool` folder and start the Tutorial Tool dev server: `npm run start`. This will *not* open a browser window.
```